src/Sulu/Bundle/ContactBundle/Admin/ContactAdmin.php

```diff
@@ -61,6 +61,43 @@ public function __construct(SecurityCheckerInterface $securityChecker, $title)
         $this->setNavigation(new Navigation($rootNavigationItem));
     }
 
+    public function getNavigationItemContacts(): NavigationItem
```

Comment:
Was this already there before? I would inline that code below because it is only used once, or at least make it private, so that we don't have to support it in the future.

Reply:
Was already there before..
It's public because this is also a point where developers can add something.